Sources/TSCBasic/Path.swift

```diff
@@ -561,11 +561,13 @@ private func mayNeedNormalization(absolute string: String) -> Bool {
 ///
 /// The normalization rules are as described for the AbsolutePath struct.
 private func normalize(absolute string: String) -> String {
+  #if os(Windows)
+    return string.standardizingPath
```

Comment:
The precondition and assert are in the `if` branch. That seems wrong?

Reply:
That does seem wrong... fixed.